libkbfs/kbfs_ops.go

```diff
@@ -482,7 +482,8 @@ func (fs *KBFSOpsStandard) getMaybeCreateRootNode(
 		if ops, ok := fs.ops[fb]; ok {
 			fs.log.CDebugf(ctx, "Triggering a paper prompt rekey on folder "+
 				"access due to unreadable MD for %s", h.GetCanonicalPath())
-			go ops.rekeyWithPrompt()
+			// TODO: do we really need explicit paper prompt here?
```

Comment:
Yes!  I think we do need it.  If someone tries to access the folder explicitly and a paper key is needed, we should show a prompt right away, rather than 10 minutes later.